stix_shifter/stix_translation/stix_translation.py

```diff
@@ -73,8 +64,10 @@ def translate(self, module, translate_type, data_source, data, options={}, recur
         
         try:
             try:
-                translator_module = importlib.import_module(
-                    "stix_shifter_modules." + module + ".stix_translation." + module + "_translator")
+                connector_module = importlib.import_module("stix_shifter_modules." + module + ".entry_point")
+                entry_point = connector_module.EntryPoint(options=options)
```

Comment:
we could still call this `interface`. just my opinion.

Reply:
no interface anymore, all interface files are delete in next commits